webpack/assets/javascripts/react_app/components/common/forms/ForemanNumericInput.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import NumericInput from './NumericInput';
+
+import { noop } from '../../../common/helpers';
+
+const ForemanNumericInput = ({
+  label,
+  className,
+  value,
+  rawValue,
+  onChange,
+  format,
+  parser,
+  step,
+  precision,
+  minValue,
+  disabled,
+  readOnly,
+  name,
+  id,
+}) => (
+  <>
+    <NumericInput
+      value={value}
+      className={className}
+      id={id}
+      format={format}
+      parser={parser}
+      onChange={onChange}
+      label={label}
+      step={step}
+      precision={precision}
+      minValue={minValue}
+      readOnly={readOnly}
+      disabled={disabled}
+    />
+    <input type="hidden" name={name} value={rawValue} />
```

Comment:
If this is the only reson we are creating new component is it strictly necessary?